engine/client/vid_common.c

```diff
@@ -1006,6 +1011,28 @@ static void R_CheckVBO( void )
 	r_bump = Cvar_Get( "r_bump", def, flags, "enable bump-mapping (r_vbo required)" );
 }
 
+/*
+===============
+R_initStrobe
+
+register strobe cvar
+===============
+*/
+_inline void R_StrobeInit( void )
+{
+	r_strobe = Cvar_Get( "r_strobe", "0", CVAR_ARCHIVE, "black frame replacement interval" );
+	r_strobe_swapinterval = Cvar_Get( "r_strobe_swapinterval", "0", CVAR_ARCHIVE, "swapping phase interval" );
+	r_strobe_debug = Cvar_Get( "r_strobe_debug", "0", CVAR_ARCHIVE, "show strobe debug information" );
+	r_strobe_cooldown = Cvar_Get( "r_strobe_cooldown", "3", CVAR_ARCHIVE, "strobe cooldown perios in secs" );
+
+	StrobeInfo.pCounter = 0; StrobeInfo.pBCounter = 0; StrobeInfo.pNCounter = 0;
```

Comment:
You may use Q_memset() here. Also global variables is always zero initialized. It's a standard too.